tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+        #######################################################################
+        # Traffic configuration Background data
+        #######################################################################
+        background_dscp = Priority(Dscp(phb=FieldPattern(choice=bg_dscp_list)))
+        background_flow = Flow(
+            name=background_flow_name,
+            tx_rx=TxRx(data_endpoint),
+            packet=[
+                Header(choice=EthernetHeader()),
+                Header(choice=Ipv4Header(priority=background_dscp))
+            ],
+            size=Size(frame_size),
+            rate=Rate('line', background_line_rate),
+            duration=Duration(FixedSeconds(seconds=traffic_duration, delay=delay, delay_unit='nanoseconds'))
+        )
+        config.flows.append(background_flow)
+
+        #######################################################################
+        # Traffic configuration Pause
+        #######################################################################
+        pause_endpoint = PortTxRx(tx_port_name='Rx', rx_port_names=['Rx'])
+        if (pause_frame_type == 'priority') :
+            p = ['0' if str(x) in test_dscp_list else 'ffff' for x in range(8)]
+            v = calculate_priority_vector(p) 
+            pause = Header(PfcPause(
+                dst=FieldPattern(choice='01:80:C2:00:00:01'),
+                src=FieldPattern(choice='00:00:fa:ce:fa:ce'),
+                class_enable_vector=FieldPattern(choice=v),
+                pause_class_0=FieldPattern(choice=p[0]),
+                pause_class_1=FieldPattern(choice=p[1]),
+                pause_class_2=FieldPattern(choice=p[2]),
+                pause_class_3=FieldPattern(choice=p[3]),
+                pause_class_4=FieldPattern(choice=p[4]),
+                pause_class_5=FieldPattern(choice=p[5]),
+                pause_class_6=FieldPattern(choice=p[6]),
+                pause_class_7=FieldPattern(choice=p[7]),
+            ))
+
+            pause_flow = Flow(
+                name='Pause Storm',
+                tx_rx=TxRx(pause_endpoint),
+                packet=[pause],
+                size=Size(64),
+                rate=Rate('line', value=100),
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        elif (pause_frame_type == 'global') :
+            pause = Header(EthernetPause(
+                dst=FieldPattern(choice='01:80:C2:00:00:01'),
+                src=FieldPattern(choice='00:00:fa:ce:fa:ce')
+            ))
+
+            pause_flow = Flow(
+                name='Pause Storm',
+                tx_rx=TxRx(pause_endpoint),
+                packet=[pause],
+                size=Size(64),
+                rate=Rate('line', value=pause_line_rate),
+                duration=Duration(Continuous(delay=0, delay_unit='nanoseconds'))
+            )
+        else :
+            pass   
+
+        config.flows.append(pause_flow)
+
+    return one_hundred_gbe
```

Comment:
It is a little bit weird to return an input argument (though I understand your point)